README.md

```diff
@@ -99,10 +98,24 @@ The format of the file is as follows (note that, due to the presence of comments
 	// Private key to use for built-in SSH server.
 	"hostkey": "hostkey",
 
-	// Authorized keys to use for authenticating users. An important note
-	// is that the comment (the part after the key itself in an entry)
-	// will	be used as name for the user internally.
-	"authkeys": "authkeys",
+	// Authorized keys to use for authenticating users.
+	"users": [
+		{
+			"type": "ssh-ed25519",
```

Comment:
I believe this field is unnecessary, as the PEM blob itself contains the key type. We could leave the user list as simply a user/publicKey key/value pair, but then again, we might want per-user options later...

Maybe kill the type field, but leave the users as objects?